docs/cli-client/bank/token-stats.md

```diff
@@ -28,21 +28,20 @@
 
 | Name,shorthand        | Default        | Description                                 | Required | Type   |
 | --------------------- | -------------- | ------------------------------------------- | -------- | ------ |
-| -e, --encoding string | hex            | String   Binary encoding (hex \|b64 \|btc ) | False    | String |
-| --home string         | /root/.iriscli | Directory for config and data               | False    | String |
-| -o, --output string   | text           | Output format (text \|json)                 | False    | String |
+| -e, --encoding | hex            | String   Binary encoding (hex\b64\btc ) | False    | String |
+| --home         | /root/.iriscli | Directory for config and data               | False    | String |
+| -o, --output    | text           | Output format (text\json)                 | False    | String |
 | --trace               |                | Print out full stack trace on errors        | False    |        |
```

Comment:
Remove all the global flag to `docs/cli-client/README.md`.

Reply:
fixed